editors/lite-xl/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           app 1.0
 PortGroup           meson 1.0
 PortGroup           legacysupport 1.1
 
-github.setup        lite-xl lite-xl 2.1.1 v
+github.setup        lite-xl lite-xl 2.1.2 v
 github.tarball_from archive
 revision            1
```

Comment:
Would you mind setting the revision to 0 with the new version? Thanks.

Reply:
Ports should not, generally, download things while building. If you need to build some subproject, it should be taken from another port *or* it should be downloaded as part of the fetch phase.